src/js/view/controls/controls.js

```diff
@@ -362,6 +362,13 @@ export default class Controls {
 
     disable(model) {
         const { nextUpToolTip, settingsMenu, infoOverlay } = this;
+        const volumeTooltip = this.controlbar.elements.volumetooltip;
+
+        
+        if (volumeTooltip) {
+            volumeTooltip.destroy();
+        }
```

Comment:
Can we create a `destroy` method for `controlbar` which destroys it's elements? This should go above the `rightClickMenu.destroy` block.